tests/net/npf/src/main.c

```diff
@@ -306,14 +306,13 @@ static void test_npf_eth_mac_addr_mask(void)
 	zassert_true(npf_remove_all_recv_rules(), "");
 }
 
-void test_main(void)
+ZTEST(net_pkt_filter_test_suite, test_net_pkt_filter)
 {
-	ztest_test_suite(net_pkt_filter_test,
-			 ztest_unit_test(test_npf_iface),
-			 ztest_unit_test(test_npf_example1),
-			 ztest_unit_test(test_npf_example2),
-			 ztest_unit_test(test_npf_eth_mac_address),
-			 ztest_unit_test(test_npf_eth_mac_addr_mask));
-
-	ztest_run_test_suite(net_pkt_filter_test);
+	test_npf_iface();
+	test_npf_example1();
+	test_npf_example2();
+	test_npf_eth_mac_address();
+	test_npf_eth_mac_addr_mask();
```

Comment:
Could those tests be also executed as separate test cases?

Reply:
only test_npf_example1 and test_npf_example2 can be split out.
I'll change it.